### DISCUSSION TITLE
Migration from DependencyTrack

### DISCUSSION THREAD

**Discussion by tvannahl** [DependencyTrack/hyades#1270] [Q&A] [answered]
Moin,

I do love this initiative and only found out about it due to the PR of pushing the chart into the different repo. As a users of DependencyTrack in Kubernetes I was wondering if there is a migration path towards hyades already.

And do I assume correctly that Hyades does not keep parity with the Database Schema of DependencyTrack?

kind regards,
Theo

**Comment by nscuro** [comment, marked as answer]
Hi, Hyades is not yet ready for production use by the general public, but we're working towards it: https://github.com/DependencyTrack/hyades/issues/860

Migration paths from existing Dependency-Track v4.x deployments are part of our GA roadmap. Most likely there'll be some kind of migration tooling that handles schema changes etc. automatically.

We talked about Hyades in some of our community meetings, you can find the slides and links to recordings here: https://github.com/DependencyTrack/community

**Reply by tvannahl** [reply]
Thank you, this has been most helpful 🙏
